src/lib/sensor_calibration/Accelerometer.cpp

```diff
@@ -286,11 +292,22 @@ bool Accelerometer::ParametersSave()
 
 void Accelerometer::PrintStatus()
 {
-	PX4_INFO("%s %" PRIu32 " EN: %d, offset: [%.4f %.4f %.4f], scale: [%.4f %.4f %.4f], %.1f degC",
-		 SensorString(), device_id(), enabled(),
-		 (double)_offset(0), (double)_offset(1), (double)_offset(2),
-		 (double)_scale(0), (double)_scale(1), (double)_scale(2),
-		 (double)_temperature);
+	if (external()) {
```

Comment:
Does this one get called for uncalibrated sensors as well?
Just so that we continue to have a quick way to inspect which sensors are external, indep from calib state.

Reply:
Yes it's set after the device id is set (typically construction). The output is in `sensors status` (which could use a bit of a cleanup).

Reply:
We could also try to squeeze some indicator of external bus into the device id string (listener, etc).